config/nightwatch.docker-compose.js

```diff
@@ -19,17 +16,23 @@ module.exports = {
   parallel_process_delay: 10,
   disable_colors: process.env.BUILDTYPE === 'production',
 
+  webdriver: {
+    start_process: true,
+    server_path: process.env.CHROMEDRIVER_FILEPATH || chromedriver.path,
+    port: 9515,
+    status_poll_interval: 1000,
+    max_status_poll_tries: 5,
```

Comment:
The default values were 100 and 5, which made the timeout for the ChromeDriver setup a little too strict, causing it to fail before the tests even started.